connectors/sources/dropbox.py

```diff
@@ -1146,20 +1192,19 @@ async def add_document_to_list(self, func, account_id, folder_id, is_shared=Fals
     async def fetch_file_folders_with_dls(self):
         account_id, member_id = await self.get_account_details()
         self.dropbox_client.member_id = member_id
-        async for folder_id in self.get_team_folder_id():
-            async for mapped_document in self.add_document_to_list(
-                func=self._fetch_files_folders,
-                account_id=account_id,
-                folder_id=folder_id,
-            ):
-                yield mapped_document
-            async for mapped_document in self.add_document_to_list(
-                func=self._fetch_shared_files,
-                account_id=account_id,
-                folder_id=folder_id,
-                is_shared=True,
-            ):
-                yield mapped_document
+        async for mapped_document in self.add_document_to_list(
+            func=self._fetch_files_folders,
+            account_id=account_id,
+            folder_id=self.dropbox_client.member_id,
```

Comment:
Why are we using the `self.dropbox_client.member_id` here instead of the original `folder_id`?

Reply:
Seems like these changes does not change anything, i tried to revert the changes and tested the connector; it also works properly. So, reverting this changes to reduce confusion :)